apps/mobile/src/components/TabNavivation/TabBar.tsx

```diff
@@ -22,21 +45,36 @@ export const BottomNavigationTabBar: FC<BottomTabBarProps> = ({
 		});
 
 		if (!focused && !event.defaultPrevented) {
+			appActions.setActiveWidgetRoute(route.name);
 			navigation.navigate({ name: route.name, merge: true } as never);
 		}
 	};
 
+	const drawerStatus = useDrawerStatus();
+
 	return (
 		<View style={[styles.container, containerStyle]}>
+			<NavigationItem
+				route={{
+					key: 'OurProject',
+					name: 'OurProject',
+					params: {} as never,
+				}}
```

Comment:
did not seems that you've completed implement this piece - should not be static I guess

Reply:
Following the design, we have a Walless icon used to toggle the drawer and it is placed in the bottom tabs such as the other tabs, but it does not has a tab behavior. That's why I let it empty, do you have any suggestions?